jit/v2/JIT_v2.0.0.yaml

```diff
@@ -2086,14 +2086,47 @@ components:
     #######
     # Moves
     #######
+    MovesList:
+      type: array
+      description: |
+        An array of `Moves`. Can be per VSA-Partner or as an accumulated result.
+      items:
+        $ref: '#/components/schemas/Moves'
     Moves:
       type: object
       title: Moves
       description: |
         Forecasted container moves for loading and discharging, empty or laden, at a terminal, covering various types such as Dry, Reefer, Out-of-Gauge (OOG), and Restows. These can be detailed at varying levels of granularity.
 
         **Condition:** Can only be provided if `portCallServiceType` is `MOVES`
+
+        **Condition:** `carrierCode` and `carrierCodeListProvider` are to be used if the moves provided in this object is **only** for a single VSA-Partner. If this object is provided **without** `carrierCode` and `carrierCodeListProvider` then it means it is an aggregated result for the remaining moves.
```

Comment:
I propose to add examples: 
**Multiple Moves Objects with Carrier Codes**
If `carrierCode` and `carrierCodeListProvider` are provided for multiple Moves objects, any object without these fields represents the accumulated result of moves for the remaining VSA partners in the vessel.
**Single Moves Object Without Carrier Codes**
If only one Moves object exists and does not include `carrierCode` and `carrierCodeListProvider`, it represents the accumulated result for all VSA partners on the vessel.
**Single Moves Object with Carrier Codes**
If only one Moves object exists and includes `carrierCode` and `carrierCodeListProvider`, it represents moves specifically for the indicated carrier.